.dockerignore

```diff
@@ -1,3 +1,6 @@
 .git
-.github
-.env.example
\ No newline at end of file
+.env.example
+**/.github
+**/node_modules
+**/.next
```

Comment:
Why are we doing `**/<folder>` here? Wouldn't simply `<folder>` do the same thing? 
[Ref](https://docs.docker.com/build/building/context/#dockerignore-files)

Reply:
I don't think that's true. Dockerignore works differently to gitignore and is a bit unintuitive because of that:

https://shisho.dev/blog/posts/how-to-use-dockerignore/#:~:text=Syntax%20difference%20between%20.dockerignore%20and%20.gitignore

All paths are relative to the root

Reply:
I see. Surprising that this isn't mentioned in the official docker docs. Thank you for the reference.